design/lvm/resize_workflow.md

```diff
@@ -131,6 +131,7 @@ Below is high level sequence diagram for volume expansion workflow
 
 ### Test Plan
 A test plan will include following test cases:
+<<<<<<< HEAD
```

Comment:
@mittachaitu -- can you check this please.

Reply:
Updated.. Missed while resolving comments